docs/pallas/tpu/core_map.md

```diff
@@ -0,0 +1,358 @@
+# Pallas Core-specifc Programming
+
+In this guide, we explore using `pl.core_map` to write Pallas kernels. Compared with `pallas_call`, `core_map` offers a few key characteristics:
+
+* **Per-core level programming**: You write code for an TPU/GPU core, not for a JAX device. This is crucial if you want to specifically control a core, or how cores communicate and distribute work among one another.
+
+* **Flexible pipelining**: You have the option to write pipelining communications on your own, instead of relying on Pallas grids and specs. This is helpful if your pipeline diverges from the standard "copy-in, compute & copy-out" pattern.
```

Comment:
We can already do this in `pallas_call`, by simply not using the grid.

Reply:
True, since deep down `core_map` just uses `pallas_call`. Thanks for the clarification.

I am thinking maybe I should frame it clearly that `core_map` is a wrapper of `pallas_call` minus the grid-pipelining feature, with a clear awareness of cores.